bin/dotr

```diff
@@ -29,9 +29,13 @@ function runMain {
     fi
 }
 
+first_arg=$1
+
 if [ -z "$1" ]; then
     echo "Starting dotty REPL..."
     eval "$DOTTY_ROOT/bin/dotc -repl"
+elif [[ ${first_arg:0:1} == "-" ]]; then
+    eval "$DOTTY_ROOT/bin/dotc -repl $@"
```

Comment:
Sophisticated enough for now - we will have to revisit this script to get it on par with the `scala` script eventually.